log4j-nosql/src/main/java/org/apache/logging/log4j/nosql/appender/mongodb/MongoDbProvider.java

```diff
@@ -97,6 +103,8 @@ public String toString() {
     @PluginFactory
     public static MongoDbProvider createNoSqlProvider(
             @PluginAttribute("collectionName") final String collectionName,
+            @PluginAttribute(value = "capped", defaultBoolean = false) final Boolean isCapped,
```

Comment:
I took a look at the CassandraAppender. It's not a big deal to adjust the MongoDbProvider to match this style. But the API is also broken. Or do you intend to keep the old public methods? Shouldn't this even be another issues and pull-request?